### PR TITLE
Free Distribution Testing

### PR DESCRIPTION
**Pull request by ayushrana182**
PR for Free Distribution Testing

### CODE REVIEW

File: doc/source/free-distribution-testing.md [hunk 1]
```diff
@@ -0,0 +1,437 @@
...
+
+`35.5 > 5`
+
+The alternative hypothesis is then proved and therefore the null hypothesis is rejected. In this way we conclude that Me1 > Me2, which implies that the median of the defective production after the wage incentive, is definitely less than the median before the increase. This means that the task of lowering the production of defective units was achieved by raising the salary of its employees.
+
+## About the Author
+
+Graduated in Mechanical Engineering, and a master’s degree in teaching component, I gave classes in several institutes of mathematics and physics, but I also dedicated several years of my life as a television producer, I did the scripts for mikes, the camera direction, editing of video and even the location. Later I was dedicated to SEO writing for a couple of years. I like poetry, chess and dominoes.
+
+
```
Comment (jbampton) [inline doc/source/free-distribution-testing.md:418]: Too many extra lines

**Review by jbampton — CHANGES_REQUESTED** [review]

File: doc/source/free-distribution-testing.md (hunk 1 above)
Comment (ayushrana182) [reply doc/source/free-distribution-testing.md]: Sorry, about that. I have resolved it on my latest commit **Removed White Space from File**

Comment (ayushrana182) [reply doc/source/free-distribution-testing.md]: sorry about that,  resolved it in my recent commit.

File: doc/source/index.rst [hunk 2]
```diff
@@ -50,6 +50,7 @@ Welcome to slurp's documentation!
    fermat.md
    fibonacci.md
    frequentist.md
+   free-distribution-testing.md
```
Comment (jbampton) [inline doc/source/index.rst:53]:
```suggestion
   free-distribution-testing.md
   frequentist.md
```

File: doc/source/free-distribution-testing.md [hunk 3]
```diff
@@ -0,0 +1,415 @@
...
+This test is used when the critical area or rejection region is at both ends of the distribution and the idea is to determine if the sample we are analyzing is within or outside of a certain range of values. On the other hand, instead of accepting the null hypothesis, the alternative hypothesis is accepted, when the value is within the two tails in the probability distribution. In this case α is divided into two equal parts and therefore the critical value will be α / 2. In general, the value of α = 0.05 is established so that the critical region will be found on both sides, but with a value of 0.05 / 2 = 0.025. Both sides are considered in order to verify if the estimated parameter is greater or less than the assumed parameter, so that in this case, the extreme values act as evidence against the null hypothesis.
+
+![Two-tail test ](_static/images/free-distribution-testing/chart3.jpg)
+
+Hypothesis tests for a population median
+
+|            |      Left Tail Test      | Right tail test | Two Tail Test |
+| :--------: | :----------------------: | :-------------: | ------------- |
+| Hypothesis | `Ho: Median ≥ 〖Med〗_0`
```
Comment (jbampton) [inline doc/source/free-distribution-testing.md:71]:
```suggestion
| Hypothesis | `Ho: Median ≥ 〖Med〗_0`
```

File: doc/source/free-distribution-testing.md [hunk 4]
```diff
@@ -0,0 +1,415 @@
...
+Now we are going to assign the value of x, which, as we saw previously, is the one with the least amount among the 2 signs, so x = 2 since this is the least value between the numbers 6 and 2.
+We must calculate the value of P for all those values less than or equal to the value of x since the sum goes from 0 to x = 2, that is, P (x ≤ 2) = P (x = 0) + P (x = 1 ) + P (x = 2) and remember that p = q = 0.5, substituting the data in the formula we have:
+
+`P= ∑_(k=0)^X▒〖nCkp^k 〗 p^(n-k)`
+`P = 8C0〖(0,5)〗^0 〖(0,5)〗^((8-0)) + 8C1〖(0,5)〗^1 〖(0,5)〗^((8-1)) + 8C2〖(0,5)〗^2 〖(0,5)〗^((8-2))`
+`P = 8C0(0,5)^8 + 8C1(0,5)^1 (0,5)^7 + 8C2(0,5)^2 (0,5)^6`
+Recall the combination formula `NCn = N!/(N-n)!n!` substituting the values in the formula we have:
+`P= 8!/8!0! 〖(0,5)〗^8 + 8!/7!1! 〖(0,5)〗^1 〖(0,5)〗^7 + 8!/6!2! 〖(0,5)〗^2 〖(0,5)〗^6`
```
Comment (jbampton) [inline doc/source/free-distribution-testing.md:147]:
```suggestion
`P= 8!/8!0! 〖(0,5)〗^8 + 8!/7!1! 〖(0,5)〗^1 〖(0,5)〗^7 + 8!/6!2! 〖(0,5)〗^2 〖(0,5)〗^6`
```

**Review by jbampton — COMMENTED** [review]
https://github.com/slurpcode/slurp/runs/4976094817?check_suite_focus=true.

**Pull request merged**